src/Administration/Resources/app/administration/src/module/sw-cms/component/sw-cms-sidebar/index.js

```diff
@@ -132,9 +133,29 @@ Component.register('sw-cms-sidebar', {
             return this.pageConfigErrors.length > 0;
         },
 
+        cmsBlocksBySelectedBlockCategory() {
+            const result = Object.values(this.cmsBlocks).filter(b => b.hidden !== true);
+
+            if (this.currentBlockCategory === 'favorite') {
+                return result.filter(b => this.cmsBlockFavorites.isFavorite(b.name));
+            }
+
+            return result.filter(b => b.category === this.currentBlockCategory);
+        },
+
+        cmsBlockFavoritesTypes() {
+            return this.cmsBlockFavorites.getFavoriteBlockType();
+        },
+
         ...mapPropertyErrors('page', ['name']),
     },
 
+    created() {
+        if (this.blockTypes.some(blockName => this.cmsBlockFavorites.isFavorite(blockName))) {
+            this.currentBlockCategory = 'favorite';
+        }
```

Comment:
For the sake of extensibility, please do not put content into "created" besides a single method call `createdComponent();` and proceed with your code in that method.